### DISCUSSION TITLE
RuntimeError: status command squeue fails to execute.job_id:Submitted batch job 9399  error message:squeue: error: Invalid job id: Submitted

### DISCUSSION THREAD

**Discussion by maverick422** [deepmodeling/dpgen#1441] [Q&A]
sorry to bother here is the detail
dpgen 0.12.0 dp2.2.7
here is the log file 
[problem.txt](https://github.com/deepmodeling/dpgen/files/13810954/problem.txt)
and parm.json
[param.json](https://github.com/deepmodeling/dpgen/files/13810966/param.json)
and machine.json
[machine.json](https://github.com/deepmodeling/dpgen/files/13810968/machine.json)

**Comment by njzjz** [comment]
Does your Slurm program support [`--parsable`](https://slurm.schedmd.com/sbatch.html#OPT_parsable)?
